frontend/src/components/auth/saml.tsx

```diff
@@ -0,0 +1,37 @@
+"use client"
+
+import { ComponentPropsWithoutRef, useState } from "react"
+
+import { authConfig } from "@/config/auth"
+import { Button } from "@/components/ui/button"
+import { Icons } from "@/components/icons"
+
+type SamlSSOButtonProps = ComponentPropsWithoutRef<typeof Button>
+export function SamlSSOButton(props: SamlSSOButtonProps) {
+  const [isLoading, setIsLoading] = useState<boolean>(false)
+  const handleClick = async () => {
+    try {
+      setIsLoading(true)
+      window.location.href = authConfig.samlAuthorizationUrl
```

Comment:
Should we not serve this from the backend, and allow this to be configured from within the app? I see that this points to a backend url, should we not just hit this endpoint instead (we have autogenerated client code for this)? It made sense to redirect to the IDP when we were using ssoready, but we're now redirecting the page to the backend url